kratos/geometries/geometry.h

```diff
@@ -389,6 +385,218 @@ class Geometry : public PointerVector<TPointType>
         return *this;
     }
 
+     operator PointsArrayType&()
```

Comment:
@pooyan-dadvand this is the operator that we discussed this morning in order to make it compatible with the rest of the code